src/vcpkg/build.cpp

```diff
@@ -752,6 +752,16 @@ namespace vcpkg::Build
             {"_VCPKG_NO_DOWNLOADS", !Util::Enum::to_bool(action.build_options.allow_downloads) ? "1" : "0"},
         };
 
+        if (action.build_options.download_tool == DownloadTool::ARIA2)
+        {
+#if defined(_WIN32) | defined(__APPLE__)
+            variables.push_back({"ARIA2", paths.get_tool_exe(Tools::ARIA2)});
```

Comment:
get_tool_exe should be finding the tool on the PATH and always passing it in. For instance, here's what we do for git:

```
billyoneal@Billys-MacBook-Pro vcpkg % ./vcpkg fetch git   
/usr/bin/git
```

Reply:
https://github.com/BillyONeal/vcpkg-tool/commit/3fb9e61b217b48b1720a4dde75c982983116e014 may work?